### ISSUE TITLE
Modification

### ISSUE THREAD

**Issue by ndcuong91** [titikid/tensorflow_slim#1]
### Modification list from original code:
   * [x]  Eval in cpu that can be run in parallel with training
   * [x]  resume training from folder
   * [x]  Customize log display
   * [ ]  [Classic cycle for train + val](https://stackoverflow.com/questions/49905773/how-to-log-validation-loss-and-accuracy-using-tfslim)
   * [ ]  Save log
   * [ ]  Save best accuracy model
   * [ ]  Save hyper-params when training
